saleor/graphql/account/types.py

```diff
@@ -41,20 +42,28 @@ def resolve_country(self, info):
 
 
 class User(CountableDjangoObjectType):
-    permissions = graphene.List(
-        PermissionDisplay, description='List of user\'s permissions.')
     addresses = gql_optimizer.field(
-        graphene.List(
-            Address, description='List of all user\'s addresses.'),
+        graphene.List(Address, description='List of all user\'s addresses.'),
         model_field='addresses')
     note = graphene.String(description='A note about the customer')
+    orders = gql_optimizer.field(
+        PrefetchingConnectionField(
+            'saleor.graphql.order.types.Order',
+            description='List of user\'s orders.'),
+        model_field='orders')
+    permissions = graphene.List(
+        PermissionDisplay, description='List of user\'s permissions.')
```

Comment:
```suggestion
        PermissionDisplay, description='The permission list of the user.')
```

Reply:
I'd leave the descriptions unchanged for now. They need to be retaken anyway in a dedicated PR probably. Also, I think that saying "List of something.." is quite common in our API.